estuary/utils/general.py

```diff
@@ -196,39 +196,39 @@ def get_correlated_nodes_count(results, reverse=False):
         raise RuntimeError('This function can\'t be called with one or zero elements')
 
     correlated_nodes = []
-    if not reverse:
+    if not forward:
```

Comment:
Could you swap the code blocks in the if and else and change the if statement to `if forward:`? It'd make it more readable.